studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx

```diff
@@ -132,123 +132,130 @@ const TaxID = () => {
-                  >
-                    Cancel
-                  </Button>
-                  <Button
-                    type="primary"
-                    htmlType="submit"
-                    loading={isUpdating}
-                    disabled={!hasChanges || isUpdating}
-                    onClick={() => onSaveTaxIds()}
-                  >
-                    Save
-                  </Button>
-                </div>
-              </div>
-            )
-          }
-        >
-          <Panel.Content className="w-8/12 space-y-4">
-            {taxIdValues.length >= 1 ? (
-              <div className="w-full space-y-2">
-                {taxIdValues.map((taxId: StripeTaxId, idx: number) => {
-                  const selectedTaxId = TAX_IDS.find((option) => option.name === taxId.name)
-
-                  return (
-                    <div key={`tax-id-${idx}`} className="flex items-center space-x-2">
-                      <Listbox
-                        value={selectedTaxId?.name}
-                        onChange={(e: any) => onUpdateTaxId(taxId.id, 'name', e)}
-                        disabled={!canUpdateTaxIds}
-                        className="w-[200px]"
-                      >
-                        {TAX_IDS.sort((a, b) => a.country.localeCompare(b.country)).map(
-                          (option) => (
-                            <Listbox.Option
-                              key={option.name}
-                              value={option.name}
-                              label={option.name}
-                            >
-                              {option.country} - {option.name}
-                            </Listbox.Option>
-                          )
-                        )}
-                      </Listbox>
-                      <Input
-                        value={taxId.value}
-                        placeholder={selectedTaxId?.placeholder ?? ''}
-                        onChange={(e: any) => onUpdateTaxId(taxId.id, 'value', e.target.value)}
-                        disabled={!canUpdateTaxIds}
-                      />
-                      {canUpdateTaxIds && (
-                        <Button
-                          type="text"
-                          icon={<IconX />}
-                          onClick={() => onRemoveTaxId(taxId.id)}
-                        />
-                      )}
-                    </div>
-                  )
-                })}
-              </div>
-            ) : (
-              <div>
-                <p className="flex items-center space-x-2 text-sm text-foreground-lighter">
-                  No tax IDs
-                </p>
+    <ScaffoldSection>
+      <ScaffoldSectionDetail>
+        <div className="sticky space-y-2 top-12">
+          <p className="text-base m-0">Tax ID</p>
+          <p className="text-sm text-foreground-light pr-4 m-0">
+            If you would like to include specific tax ID(s) to your invoices. Old invoices are not
```

Comment:
this isn't part of the clean up but reading back, copy can be improved a bit here haha (more active voice)
```suggestion
            Add tax ID(s) to have them appear in your invoices. Old invoices are not
```